databuilder/databuilder/models/dashboard/dashboard_chart.py

```diff
@@ -141,6 +148,55 @@ def _create_record_iterator(self) -> Iterator[RDSModel]:
 
         yield record
 
+    def create_next_atlas_entity(self) -> Union[AtlasEntity, None]:
+        try:
+            return next(self._atlas_entity_iterator)
+        except StopIteration:
+            return None
+
+    def create_next_atlas_relation(self) -> Union[AtlasRelationship, None]:
+        return None
+
+    def _create_next_atlas_entity(self) -> Iterator[AtlasEntity]:
+        # Chart
+        attrs_mapping = [
+            (AtlasCommonParams.qualified_name, self._get_chart_node_key()),
+            ('name', self._chart_name),
+            ('type', self._chart_type),
+            ('url', self._chart_url),
+        ]
+
+        chart_entity_attrs = dict()
+        for attr in attrs_mapping:
```

Comment:
as you use the same approach for every Model, in the spirit of DRY maybe it would make sense to have function handling this in AtlasSerializable (and just provide attrs_mapping to it) ?